docs/Developers/Writing_Good_Commits.rst

```diff
@@ -115,7 +115,7 @@ Example:
 
 ::
 
-    Fixes https://github.com/coala/coala/issues/269
+    Closes https://github.com/coala/coala/issues/269
```

Comment:
No it's required - it's splitting up the original example given in line 11-18 into three parts and explaining each part by requoting it.